REQUIRE

```diff
@@ -1,2 +1,2 @@
-julia 0.4
-Compat
+julia 0.5
+Compat 0.18
```

Comment:
0.9.5 for take!, and it doesn't need the at-compat macro since it's not a syntax tranformation